### PR TITLE
[IMP][7.0] stock available reserved quantity

### PR DESCRIPTION
**Pull request by llacroix**

### CODE REVIEW

File: stock_available_reserved_quantity/models/stock_production_lot.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        for lot in self.browse(cr, uid, ids, context=context):
+            if lot.retained_stock < 0:
+                return False
+
+        return True
+
+    _inherit = "stock.production.lot"
+
+    _columns = {
+        'retained_stock': fields.float(string='Retained stock'),
```
Comment (joaoalf) [inline stock_available_reserved_quantity/models/stock_production_lot.py]: @llacroix For float fields it is good to add a precision.

Comment (llacroix) [inline stock_available_reserved_quantity/models/stock_production_lot.py]: Which precision should we add?

Comment (joaoalf) [inline stock_available_reserved_quantity/models/stock_production_lot.py]: @llacroix 
Ex: 

``` python
import openerp.addons.decimal_precision as dp
...
'product_qty': fields.float('Quantity', digits_compute=dp.get_precision('Product Unit of Measure'), ...)
```

**Comment by llacroix** [comment]
It would be nice to see code coverage. Any idea what happened? I don't see coverage on PRs anymore.

**Comment by yvaucher** [comment]
I replayed last green travis jobs to see if we got coverage

**Comment by jgrandguillaume** [comment]
Hi,

Could I ask you to give a bit more explanation in the manifest ? I do not understand the exact features and purpose of this work, what is "Retained stock" ?

Thanks for your clarification.

Regards,

**Comment by llacroix** [comment]
@jgrandguillaume I'll update the manifest later today.

**Comment by clonedagain** [comment]
Hi, I hadn't noticed your PR but the idea is interesting in v7 : set a quantity on each lot and report the total reserved.

Indeed you must give more explanation in the `README`.

I'd also like you to integrate with the module  `stock_available`,which is a generic module to compute the stock available for sale : 
- add an option to install you module in  `stock_available`'s config
- inherit the compute method of the field "stock_available_immediately" to subtract the reserved quantity from the quantity available for sale.
  You can use `stock_available_immediately`, `stock_available_sale` and `stock_available_mrp` as examples.

**Pull request closed without merging**